androidgen/src/main/java/com/azure/autorest/android/model/clientmodel/AndroidProxyMethod.java

```diff
@@ -46,6 +46,7 @@ protected AndroidProxyMethod(String requestContentType, IType returnType, HttpMe
                 unexpectedResponseExceptionTypes,
                 name,
                 parameters,
+                null,
```

Comment:
At least give them the same set of `parameters` instead of `null`?

Reply:
Fixed